README.md

```diff
@@ -89,6 +89,10 @@ generate-node-address = false
 queues {
     // See next section
 }
+
+// Region and accountId which will be included in resource ids
+awsRegion = us-west-2
+awsAccountId = 000000000
```

Comment:
I would add this in a dedicated seciton. If only used for resource ids, maybe

```
resourceId {
  awsRegion = ...
  awsAccountId = ...
}
```

?

If it's also used in other places, maybe simply `aws { ... }`